panel.html

```diff
@@ -9,7 +9,10 @@
 <body>
     <h1>Methods 2023 - Panel Questions</h1>
       <ul>
-        <li>add your questions...</li>
+          <li>add your questions...</li>
+          <li>Kruize Christensen - What are some resources that we can study that would not have been taught at MSU in the field of computer science</li>
+          <li>Kruize Christensen - What are some begining jobs to look at as a Computer Science graduate that are located in State</li>
+          <li>Kruize Christensen - What makes you stand out to empolyers compared to other recent graduates</li>
```

Comment:
some spelling mistakes in here, otherwise looks good

Reply:
spelling errors are fixed